beacon-chain/powchain/service.go

```diff
@@ -141,22 +143,22 @@ func (w *Web3Service) run(done <-chan struct{}) {
 	}
 }
 
-// LatestBlockNumber is a getter for blockNumber to make it read-only.
+// LatestBlockNumber exposes a getter for blockNumber.
 func (w *Web3Service) LatestBlockNumber() *big.Int {
 	return w.blockNumber
 }
 
-// LatestBlockHash is a getter for blockHash to make it read-only.
+// LatestBlockHash exposes a getter for blockHash.
 func (w *Web3Service) LatestBlockHash() common.Hash {
 	return w.blockHash
 }
 
-// ValidatorRegistered is a getter for validatorRegistered to make it read-only.
+// ValidatorRegistered exposes a getter for validatorRegistered.
```

Comment:
Please change to IsValidatorRegistered

And 

// IsValidatorRegistered in the validator manager contract on the PoW chain.